pyproject.toml

```diff
@@ -4,10 +4,10 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "basyx-python-sdk"
-version = "0.1.0"  # Replace with your actual version
+version = "1.0.0"  # actual version
```

Comment:
The comment is not a particularly good one, as it is obvious that `version` equals the version